about.html

```diff
@@ -83,30 +84,33 @@ <h4><i class="fab fa-youtube"></i> YouTube</h4>
 
     <div class="flex-container flex-v-center">
         <div><h2>Awards</h2></div>
-        <div><img src="assets/about/trophy.gif" alt="Trophy GIF" id="trophy-gif"></div>
+        <div><p id="trophy">🏆</p></div>
     </div>
 
-<p>Our society has won many achievements and awards, including: </p>
-<div id="awards">
-    <p class="emoji-bullet">🥇 Society of the Year (2016/17)</p>
-    <p class="emoji-bullet">🎖 Outstanding Event Award 2017, HackTheMidlands</p>
-    <p class="emoji-bullet">🏅 Inspirational Committee Member 2016, Jack Wearden</p>
-    <p class="emoji-bullet">👩‍💻 Outstanding Event Award 2016, the Women in Technology Conference with WISE</p>
-    <p class="emoji-bullet">🏆 EPS Community Award 2019, Tom Goodman</p>
-</div>
+    <p>Our society has won many achievements and awards, including: </p>
+    <div id="awards">
+        <p class="emoji-bullet">🏆 The Great Achievement in EPS, Tom Goodman, CSS Secretary (2020)</p>
+        <p class="emoji-bullet">🏆 Community Award, Tom Goodman, CSS President (2019)</p>
+        <p class="emoji-bullet">🏆 Society of the Year (2017)</p>
+        <p class="emoji-bullet">🏆 Outstanding Event Award for HackTheMidlands (2017)</p>
+        <p class="emoji-bullet">🏆 Outstanding Event Award for Women In Technology Conference (2017)</p>
+        <p class="subtext">Joint award with WISE, the Women in Science & Engineering Society</p>
+        <p class="emoji-bullet">🏆 Inspirational Committee Member, Jack Wearden, CSS President (2016)</p>
+    </div>
 
 
-<h2>Constitution 📄</h2>
-<p>CSS is part of the <a href="https://www.guildofstudents.com">Guild of Students</a>. 🙌</p>
-<p>That means we've got a constitution that you can find below 👇</p>
-<ul>
-    <li><a href="https://docs.google.com/document/d/1J3_EWs3dd2gq5T_xYylcP5pDzm0HHAYCTgiqKUYwkls">Constitution — Google Doc</a></li>
-    <li><a href="constitution.pdf">Constitution — PDF</a> </li>
-</ul>
-<p>We recommend that all our members read it at least once. </p>
-<p> You can also find our agendas <a href="../agendas/">here</a>. </p>
+    <h2>Constitution 📄</h2>
+    <p>CSS is part of the <a href="https://www.guildofstudents.com">Guild of Students</a>. 🙌</p>
+    <p>That means we've got a constitution that you can find below 👇</p>
+    <ul>
+        <li><a href="https://docs.google.com/document/d/1J3_EWs3dd2gq5T_xYylcP5pDzm0HHAYCTgiqKUYwkls">Constitution —
+            Google Doc</a></li>
+        <li><a href="constitution.pdf">Constitution — PDF</a></li>
+    </ul>
+    <p>We recommend that all our members read it at least once. </p>
+    <p> You can also find our agendas <a href="../agendas/">here</a>. </p>
 
-    <h2>Contact Us ✉️</h2>
+    <div><h2>Contact Us <i class="fas fa-envelope"></i></h2></div>
```

Comment:
Not sure about the <div> container here, and the why has the email emoji changed?

Reply:
Container was left over from me trying to get FontAwesome to work, will be removed momentarily. I replaced the image as I was replacing all the emojis for the categories and this didn't have an emoji for it (That I could see at least)

Reply:
the one that was used on the webpage (atm) = ✉️